pages/doc.js

```diff
@@ -93,44 +103,76 @@ export default class Documentation extends Component {
       debug: false // Set debug to true if you want to inspect the dropdown
     })
   }
-
-  getLinkHref = (section, file) => {
-    const sectionSlug =
-      sidebar[section].slug || kebabCase(sidebar[section].name)
-    const fileSlug = file ? kebabCase(file.slice(0, -3)) : undefined
-    return `/doc/${compact([sectionSlug, fileSlug]).join('/')}`
+  getLinkHref = (section, file = null, subsection) => {
```

Comment:
unify everywhere functions and call: (section, subsection, file)

Reply:
ok. done